serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py

```diff
@@ -1,49 +1,44 @@
-# Copyright (C) 2018-2020 Intel Corporation
-#
-# SPDX-License-Identifier: MIT
-
 import os
 import numpy as np
 import sys
 from skimage.measure import find_contours, approximate_polygon
-
-# workaround for tf.placeholder() is not compatible with eager execution
-# https://github.com/tensorflow/tensorflow/issues/18165
+from pathlib import Path
 import tensorflow as tf
-tf.compat.v1.disable_eager_execution()
-#import tensorflow.compat.v1 as tf
-#   tf.disable_v2_behavior()
-
-# The directory should contain a clone of
-# https://github.com/matterport/Mask_RCNN repository and
-# downloaded mask_rcnn_coco.h5 model.
-MASK_RCNN_DIR = os.path.abspath(os.environ.get('MASK_RCNN_DIR'))
-if MASK_RCNN_DIR:
-    sys.path.append(MASK_RCNN_DIR)  # To find local version of the library
-    sys.path.append(os.path.join(MASK_RCNN_DIR, 'samples/coco'))
-
+MASK_RCNN_DIR = '/opt/nuclio/Mask_RCNN'
+sys.path.append(MASK_RCNN_DIR)  # To find local version of the library
 from mrcnn import model as modellib
-import coco
+from mrcnn.config import Config
+
 
 class ModelLoader:
+
     def __init__(self, labels):
-        COCO_MODEL_PATH = os.path.join(MASK_RCNN_DIR, "mask_rcnn_coco.h5")
```

Comment:
I used MASK_RCNN_DIR to debug the code outside of a docker container.

Reply:
How did you mount it? I don't see how this is being used! I use vscode to connect to a docker for debugging. It is great if you haven't tried it.
Nevertheless, I reverted this change.